### DISCUSSION TITLE
What does "No DSS meet criteria" mean?

### DISCUSSION THREAD

**Discussion by msteele999** [njthomson/SrvSurvey#148] [Q&A] [answered]
Hello,

I LOVE your tool and use it every time I play.

I have a question - after I complete an FSS scan and identify all objects in the system, I often get the message "No DSS meet criteria". This is in systems that have BIO signals that are greater than my threshold (1MM Credits) - what is that message referring to?

Best regards,

CMDR SoliDeoGloria

**Comment by msteele999** [comment]
I answered my own question! I had some settings that were limiting what was shown.

**Comment by njthomson** [comment, marked as answer]
Yup, these are the settings that control what bodies or rings are deemed interesting for DSS:

![image](https://github.com/user-attachments/assets/e3fb8379-33c6-4799-b8d9-00c66ce6cffe)
